### PR TITLE
bumped cats version to 0.8.1

### PR DESCRIPTION
**Pull request by Mullefa**
@adamnfish @philwills current version of `cats` used by `play-googleauth` is incompatible with the latest version of `cats` (`0.8.1`)

### CODE REVIEW

File: module/src/main/scala/com/gu/googleauth/actions.scala [hunk 1]
```diff
@@ -108,11 +108,8 @@ trait Actions extends UserIdentifier {
         }
       case None =>
         Logger.warn("Login failure, anti-forgery token missing")
-        XorT.left[Future, Result, UserIdentity] {
-          Future.successful {
-            redirectWithError(failureRedirectTarget, "Login failure, anti-forgery token missing", authConfig.antiForgeryKey, request.session)
-          }
-        }
+        val redirect = redirectWithError(failureRedirectTarget, "Login failure, anti-forgery token missing", authConfig.antiForgeryKey, request.session)
+        EitherT.fromEither(Left(redirect))
```
Comment (philwills) [inline module/src/main/scala/com/gu/googleauth/actions.scala]: Much nicer

Comment (adamnfish) [inline module/src/main/scala/com/gu/googleauth/actions.scala]: :+10:

**Comment by adamnfish** [comment]
Great change :+1:

I think it's worth putting the (compatible) cats version(s) on the README as well though, since this new version will not be compatible with old versions of cats. The same will be true in the future until cats' API is more stable.

**Comment by Mullefa** [comment]
@adamnfish is the README change ok?

**Comment by adamnfish** [comment]
Looks great!

I've just realised we didn't ever get round to releasing https://github.com/guardian/play-googleauth/pull/41 as well. Could do a release with both, or I'll follow that up later, up to you.

**Comment by Mullefa** [comment]
A release with both makes sense 👍  If you merge #41 I can rebase off that?

**Comment by Mullefa** [comment]
Done

**Pull request merged**